packages/website/src/content/docs/reference/configuration.md

```diff
@@ -155,7 +155,7 @@ import { trigger } from '@spotlightjs/spotlight';
 
 trigger('sentry:showError', {
   event: string,
-  eventId: string,
+  event_id: string,
```

Comment:
I think these docs were incorrect? We use `event_id` in the code, as far as I can see 😅

Reply:
Ugh, sorry :(